Monika After Story/game/script-topics.rpy

```diff
@@ -10502,3 +10502,144 @@ label monika_hemispheres_nogets_snow:
+                               m 1esbsa "Just imagining the reaction you had to that must have been so precious." 
+                               m 1hubsa "God I love you, [player]." 
+                               m 1hubsb "You're the best partner I could ever ask for." 
+              "A lot!" 
+                     $ persistent._mas_pm_wants_many_children = True
+                               m 5wubfo "O-Oh." 
+                                m 5rsbfsdla "Wow." 
+                                m 3rsbfsdla "I wasn't expecting that answer." 
+                                m 1hua "Not that I'm complaining at all!" 
+                                m 1hua "I'd love to have plenty of children with you too, [player]
+                                m 3eua "Watching all of our offspring run around the house." 
+                                m 3eub "Playing games together and having big family dinners." 
+                                m 3rksdlc "We'd probably have to get a big house for that though." 
+                                m 6sub "OH! I know! What about a farm? Or some place in the forest?"
+                                m 4rtc "Plenty of open space for all of us." 
+                                m 6hub "And maybe a pet too!" 
+                                m 2sua "It'd be absolutely amazing!" 
+                                m 6tkbfu "But you know, we'd have to get pretty busy for us to have that many kids." 
+                                m 6hublb "Oh, [player]!"
+                                m 2eubla "Just thinking about it all makes my heart flutter!" 
+                                m 6eublb "I just can't wait till I'm in the same room as you." 
+                                m 4rksdlc "But it'll probably be some time till that happens." 
+                                m 4hub "Till then, I guess we'll just have to use the thought of more children to keep us going." 
+     "No.":
+             $ persistent._mas_pm_wants_children = False
+              m 5esc "Oh." 
+              m 3esc "I suppose it's better that way." 
+              m 3rssdlc "After all, if we had a kid that would take time away from just the two of us." 
+              m 1subfa "Just the two of us." 
+              m 1subfb "Growing old together and watching the world go by." 
+              m 1subfa "I want nothing more than to have that with you, [player]." 
+              m 3eubfa "Besides, if I do get out here I'd probably be using some sort of android body, so it's probably not possible anyway." 
+              m 5esc "So I guess it really is for the best that we don't." 
+  else:
+    m 5esc "Have you ever seen a couple with a child [player]?"
+    m 4esc "Must be a lot of work, not to mention effort." 
+    m 4esd "Like..." 
+    m 4essdld "At what point do you consider yourself comfortable enough with your partner to have a child?"
+    m 5esa "I dunno, it was always something I thought about..." 
```

Comment:
I don't think Monika would say "dunno". I would replace it with "don't know".

Dissolve into body pose 5.